src/commands/toolchain.rs

```diff
@@ -64,12 +86,27 @@ fn name_allowed(s: &str) -> Result<String> {
     }
 }
 
+fn channel_allowed(s: &str) -> Result<String> {
```

Comment:
What is `s`?

Also a bit confused about the returning of an empty `String` version of `s`

Reply:
`channel_allowed` is used as clap `value_parser` for channel, so s means the `channel` option value. And as `--channel` is optional, we should return empty `String`, for unwrap `channel: Option<String>` in `Export`

Reply:
Would it be more apt to rename to `channel`?

Reply:
I will update it later👌

Reply:
@Braqzen Done